views/layouts/main.handlebars

```diff
@@ -1,102 +1,174 @@
-<!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
 
   <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset='UTF-8' />
+    <meta name='viewport' content='width=device-width, initial-scale=1.0' />
     <title>Elevator Pitch</title>
 
-    {{!-- Google Fonts --}}
+    {{! Google Fonts }}
     <link rel='preconnect' href='https://fonts.googleapis.com' />
     <link rel='preconnect' href='https://fonts.gstatic.com' crossorigin />
-    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700;900&display=swap' rel='stylesheet' />
-
-    {{!-- Bootstrap Styles --}}
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
-
-    {{!-- CSS Animation Helper --}}
-    <link rel='stylesheet' href='//cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css' />
-    
-    {{!-- Local styling --}}
-    <link rel="stylesheet" href="/css/bs-overrides.css" />
-    <link rel="stylesheet" href="/css/utils.css" />
-    <link rel="stylesheet" href="/css/branding.css" />
-
-    {{!-- Bootstrap Script --}}
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
-    
+    <link
+      href='https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700;900&display=swap'
+      rel='stylesheet'
+    />
+
+    {{! Bootstrap Styles/Icons }}
+    <link
+      rel='stylesheet'
+      href='https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css'
+    />
+    <link
+      href='https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css'
+      rel='stylesheet'
+      integrity='sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3'
+      crossorigin='anonymous'
+    />
+
+    {{! CSS Animation Helper }}
+    <link
+      rel='stylesheet'
+      href='//cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css'
+    />
+
+    {{! Local styling }}
+    <link rel='stylesheet' href='/css/bs-overrides.css' />
+    <link rel='stylesheet' href='/css/utils.css' />
+    <link rel='stylesheet' href='/css/branding.css' />
+
+    {{! Bootstrap Script }}
+
+    <script
+      src='https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js'
+    ></script>
+    <script
+      src='https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js'
+    ></script>
+
+    {{! jquery min script }}
+    <script
+      src='https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.min.js'
+    ></script>
```

Comment:
See my comments in Slack. We should remove the popper (included in bs bundle, and swap the order of jQuery and bootsrap. That will allow the collapsible to work as expected with no script in the view.

Reply:
All set!